src/lib.rs

```diff
@@ -255,9 +260,12 @@ fn open_browser_internal(browser: Browser, url: &str) -> Result<Output> {
         )),
     }
 }
-
-/// Open on Linux using the $BROWSER env var
-#[cfg(target_os = "linux")]
+#[cfg(any(
+    target_os = "linux",
+    target_os = "freebsd",
+    target_os = "netbsd",
+    target_os = "openbsd"
+))]
 fn open_on_linux_using_browser_env(url: &str) -> Result<Output> {
```

Comment:
Given that it's no more Linux specific, should rename the fn to `open_on_unix_using_browser_env`